src/test/java/com/mael/ttt/players/HumanPlayerTest.java

```diff
@@ -32,7 +33,7 @@ public void repeatsUntilCellIsNumber() {
 
     @Test
     public void repeatsUntilItGetsEmptyCellIndex() {
-        board.setCell(1, "X");
+        board.setCell(1, Marks.PLAYER.toString());
```

Comment:
It may be more readable to statically import the 'Marks'. Also instead of toString(), Enum's provide you with a `name()` method. This means the code would read like:

` board.setCell(1, PLAYER.name());`